dali/operators/decoder/image_decoder.cc

```diff
@@ -144,6 +144,11 @@ the GPU implementation.
 According to the libjpeg-turbo documentation, decompression performance is improved by up to 14%
 with little reduction in quality.)code",
       false)
+  .AddOptionalArg("jpeg_fancy_upsampling",
+      R"code(Make the ``mixed`` backend use the same chroma upsampling approach as the ``cpu`` one.
+
+The option corresponds to the `JPEG fancy upsampling`.)code",
```

Comment:
what is this `JPEG fancy upsampling` referring to? Perhaps it should be 
```
the JPEG fancy upsampling performed by libraries such as libjpeg-turbo
```
?

Is this something the libjpeg-turbo flavor can disable on demand?